src/devices/lcec_epocat.h

```diff
@@ -20,7 +20,7 @@
 
 #include "../lcec.h"
 
-#define LCEC_EPOCAT_PID 0x00decade
+#define LCEC_EPOCAT_PID 0x04decade
```

Comment:
Oh.  They *aren't* the same.  I missed the `4`.

Consider changing this to `LCEC_FR4000_PID` or similar.  I'm still not sure what's up with the FR4000 vs FR1000--if the FR1000 code works, then I'd probably prefer to see it cloned so that we can support both of them.

Alternately, it's probably possible to support both devices from a single `.c` file, although the 16/32 bit thing will be entertaining.

One other thing--most of the names inside of driver files should be flagged as `static` these days, which means that having 2 drivers with identical names isn't generally a problem.